environments/dap/base/buckets.tf

```diff
@@ -0,0 +1,5 @@
+resource google_storage_bucket storage_bucket {
```

Comment:
i'm finding myself questioning what we get out of dividing out resources so granularly in cases like this where there's just one resource. is consistency worth the increased number of files to reference?

Reply:
like, to me, the benefit of file splits is knowing exactly where to find a resource in a large module, but for small modules like this, it seems like it'd be easier to just keep the config in a single file.

Reply:
Good call. I'll merge this into a central "resources.tf" and we can refactor from there.